implementations/VcIssuerMock.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "VC Issuer Mock",
+  "implementation": "vc-issuer-mock-core",
+  "issuers": [{
+      "id": "https://github.com/laysakura/vc-issuer-mock",
```

Comment:
so the issuer id is used the `issuer` value on the actual VC so it needs to match the `did:key` that your issuer signs with.

Reply:
@aljones15 Did it in https://github.com/w3c/vc-test-suite-implementations/pull/53/commits/e669feadd96aadba33956ab366469dc9778c0f27